MVCDemos/Views/CheckBox/Overview.cshtml

```diff
@@ -20,15 +20,15 @@
             <div class="dx-field-label">Indeterminate</div>
             <div class="dx-field-value">
                 @(Html.DevExtreme().CheckBox()
-                    .Value(new JS("undefined"))
+                    .Value(new JS("null"))
```

Comment:
Is `new JS` really necessary here? It's used for `undefined` because this type does not exist in .net

Reply:
looks like we can use just `null`
updated